### PR TITLE
miscellaneous fixes part one

### PR DESCRIPTION
**Pull request by Vexylius**
## About The Pull Request
Attempts to fix runtimes that sometimes happen during github tests (those with hivebots, pipes.dm and the NAD).
Removes highlander too, because it's tied to NAD, that had a component "shiploving", that was removed too, because it caused a lot of runtimes and wasn't really used. Also highlander broken on shiptest too, so.

## Why It's Good For The Game
Less runtimes - better game. Simple as.

## Changelog
:cl:
fix: fixed runtimes or whatever
del: removed NAD's ship affection
del: removed highlander antagonist
/:cl:

### CODE REVIEW

**Comment by Zevotech** [comment]
amazing. you are so fucking cool. i love this.

**Comment by triplezeta** [comment]
are you fucking kidding me

File: code/modules/atmospherics/machinery/pipes/pipes.dm [hunk 1]
```diff
@@ -16,7 +16,7 @@
 /obj/machinery/atmospherics/pipe/New()
 	add_atom_colour(pipe_color, FIXED_COLOUR_PRIORITY)
 	volume = 35 * device_type
-	..()
+	. = ..()
```
Comment (Fikou) [inline code/modules/atmospherics/machinery/pipes/pipes.dm]:
```suggestion
	return ..()
```
return is preferred at the end of procs, easier to read

File: code/modules/antagonists/nukeop/equipment/nuclearbomb.dm [hunk 2]
```diff
@@ -621,7 +621,6 @@ This is here to make the tiles around the station mininuke change when it's arme
 
 /obj/item/disk/nuclear/ComponentInitialize()
 	. = ..()
```
Comment (Fikou) [inline code/modules/antagonists/nukeop/equipment/nuclearbomb.dm]: if youre gonna make a proc that just returns parent, just delete it

File: code/modules/antagonists/blob/structures/core.dm [hunk 3]
```diff
@@ -68,4 +68,3 @@
 
 /obj/structure/blob/core/ComponentInitialize()
 	. = ..()
```
Comment (Fikou) [inline code/modules/antagonists/blob/structures/core.dm]: ditto

**Comment by Fikou** [comment]
lets pray to god this works

**Pull request merged**